animals_and_tips.html

```diff
@@ -132,21 +132,21 @@ <h2>Hover on each image to discover information about them</h2>
 </div>
 <div class="container">
 	<img src="images/placeholder.png" alt="img" class="image">	
-	<div class="overlay overlayTop">
+	<div class="overlay overlayCross">

```

Comment:
Some of the overlay types had to swapped and changed to a different overlay type as each overlay corresponded to one of 3 overlay background gradient colours